legacy/tomorecon_nersc_mpi_hdf5/.ipynb_checkpoints/Containerfile-checkpoint

```diff
@@ -0,0 +1,84 @@
+# Most of this is from: https://gitlab.com/NERSC/nersc-official-images/-/blob/main/nersc/python/3.9-anaconda-2021.11/Dockerfile
```

Comment:
I'm guessing this file was added by mistake. You should probably delete it and add something to .gitignore to prevent these files.